### PR TITLE
center text

### PR DESCRIPTION
**Pull request by AjGitGot**
center text

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -4,7 +4,7 @@
 <h2>Student names</h2>
 </head>
 <body>
-<FONT COLOR= "GREEN">
```
Comment (TatianaGitUser) [inline index.html:7]: duplicate line

**Comment by AjGitGot** [comment]
center text

File: index.html (hunk 1 above)
Comment (TatianaGitUser) [inline index.html:7]: duplicate line

**Review by TatianaGitUser — COMMENTED** [review]
please, remove duplicate line

File: index.html (hunk 1 above)
Comment (AjGitGot) [reply index.html:7]: its ok

**Review by TatianaGitUser — APPROVED** [review]

**Pull request merged**